src/Moryx.Products.Management/Facades/ProductManagementFacade.cs

```diff
@@ -43,6 +48,42 @@ public void Deactivate()
 
         public string Name => ModuleController.ModuleName;
 
+        public IDictionary<string, object> Importers
+        {
+            get
+            {
+                ValidateHealthState();
+                return ProductManager.Importers.ToDictionary(i => i.Name, i => i.Parameters);
+            }
+        }
+
+        public IReadOnlyList<Type> ProductTypes
+        {
+            get
+            {
+                ValidateHealthState();
+                return TypesProvider.ProductTypes;
+            }
+        }
+
+        public IReadOnlyList<Type> RecipeTypes
+        {
+            get
+            {
+                ValidateHealthState();
+                return TypesProvider.RecipeTypes;
+            }
+        }
+
+        public IReadOnlyList<Type> ImporterTypes
```

Comment:
Importer dict and type list seem like two soltuions for the same problem. Are they not?

Reply:
Yes they are, the dict already existed, I would propose to make it Obsolete, thereby we have three quite similar properties to provide recipe, product and importer types 😊

Reply:
True, but importers are transient objects in the container. Their type alone is insufficient, especially because the parameters object is prepared by the instance.

Reply:
Ohh ok, yeah I didn't know that. Then we will just leave it as it is 👍